<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentEditor</class>
 <widget class="QWidget" name="StudentEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item row="3" column="0">
      <widget class="QPushButton" name="ExportButton">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="InsertButton">
       <property name="text">
        <string>Insert</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="UpdateButton">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Gender:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>StudentID:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Birthday:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Class:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="STDIDEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="NameEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>IDNumber:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="IDEdit"/>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="ClassEdit"/>
     </item>
     <item row="0" column="4">
      <widget class="QComboBox" name="GenderBox">
       <item>
        <property name="text">
         <string>male</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>female</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Grade:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="GradeEdit"/>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>StadO:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QComboBox" name="stdoBox">
       <item>
        <property name="text">
         <string>PO08</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PO15</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>MajorinZZU:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLineEdit" name="ZZUEdit"/>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>MajorinUDE:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLineEdit" name="UDEEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="MajorButton">
       <property name="text">
        <string>Majorselect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="REFButton">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Condition:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Operation:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QComboBox" name="OperationBox">
       <item>
        <property name="text">
         <string>LIKE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>==</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="FilterTypeBox">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Name</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>StudentID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>IDNumber</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gender</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>StadO</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MajorinZZU</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MajorinUDE</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLineEdit" name="ConditionEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
